compiler/integration-tests/scripts/codegen-verifiers.sh

```diff
@@ -21,6 +21,11 @@ contracts_dir=$self_path/../contracts
 rm -rf $contracts_dir
 mkdir $contracts_dir
 
+# Build dependency testing contract
+# Run codegen-verifier for recursion
```

Comment:
```suggestion
```

Reply:
This shouldn't apply anymore that tests are back to their original package